client/src/pages/CommunityPage/CommunityPageGroup.js

```diff
@@ -113,22 +112,102 @@ class CommunityPageGroup extends Component {
     console.log(val);
   };
 
+  commentClick = val => {
+    console.log(val);
+  };
+
+  likeClick = val => {
+    console.log(val);
+  };
```

Comment:
Are these just placeholders?

Reply:
@ZakkMan Yes, my goal with this pull was to get the base render functionality done and allow for followup issues.  Therefore, I left any placeholders I found.